client/rechallenge/transformers.js

```diff
@@ -24,6 +24,11 @@ const transformersForHtmlJS = {
     {
       name: 'add-loop-protect',
       transformer: function addLoopProtect(file) {
+        const _contents = file.contents.toLowerCase();
+        if (file.ext === 'html' && _contents.indexOf('<script>') === -1) {
```

Comment:
I think 
```js
_contents.includes('<script>')
```

is cleaner than using `.indexOf()`. It's not a huge difference but it improves readability.